test/unittests/wasm_engine_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <gtest/gtest.h>
+#include <test/utils/hex.hpp>
+#include <test/utils/wasm_engine.hpp>
+
+using namespace fizzy;
+
+TEST(wasm_engine, multiple_args_ret_i32)
```

Comment:
If the test is only for fizzy, maybe name the suite as `fizzy_engine`.

Reply:
Just pushed the fix.